spec/serializers/page_serializer_spec.rb

```diff
@@ -42,9 +43,42 @@
 
     end
 
+    context 'related links' do
+      let(:related_links) do
+        subject.related_content[:related_links].as_json
+      end
+
+      before do
+        allow(article)
+        .to receive(:related_links)
+        .with(3)
+        .and_return(pages)
+      end
+
+      context 'when has most related articles in english' do
+        let(:pages) do
+          [
+            build(:page, slug: 'first-article', label: 'First Article'),
+            build(:page, slug: 'second-article', label: 'Second Article')
+          ]
+        end
+
+        before do
+          site.label = 'en'
+        end
+
+        it 'returns english related links' do
+          expect(related_links).to eq([
+            { title: 'First Article', path: '/en/articles/first-article' },
+            { title: 'Second Article', path: '/en/articles/second-article' }
+          ])
+        end
+      end
```

Comment:
What happen when most related articles is not english? Is there a spec needed here?